_sips/minutes/2017-05-08-sip-minutes.md

```diff
@@ -0,0 +1,93 @@
+|Scala Library Changes|Adriaan Moors| **WHAT DO I PUT HERE** |
+
+Jorge Vicente Cantero was the Process Lead.
+
+
+## Date and Location
+The meeting took place on 8 May 2017 via Google Hangouts at EPFL in Lausanne, Switzerland as well as other locations.
+
+[Watch on Scala Center YouTube channel](https://youtu.be/6rKa4OV7GfM)
+
+Minutes were taken by Darja Jovanovic.
+
+## Attendees
+
+* Martin Odersky ([@odersky](https://github.com/odersky)), EPFL
+* Jorge Vicente Cantero ([@jvican](https://github.com/jvican)), Process Lead
+* Seth Tisue ([@SethTisue](https://github.com/SethTisue)), Lightbend
+* Sébastien Doeraene ([@sjrd](https://github.com/sjrd)), EPFL
+* Eugene Burmako ([@xeno-by](https://github.com/xeno-by)), EPFL
+* Iulian Dragos ([@dragos](https://github.com/dragos)), Independent
+* Adriaan Moors ([@adriaanm](https://github.com/adriaanm)), Lightbend
+
+
+## Proceedings
+### Opening Remarks
+
+**Jorge** going over the agenda, points out that the second item will be skipped because they are waiting for the prototype from the author of the proposal.
+
+### [SIP-NN - comonadic-comprehensions](http://docs.scala-lang.org/sips/comonadic-comprehensions.html)
+[YouTube time: 1:39](https://youtu.be/6rKa4OV7GfM?t=99)
+
+Proposal aims to introduce new syntax from comprehension for monads to comonads.
+Martin is the reviewer. He asks others attendees for their opinion on this.  Everyone had read the SIP.
+
+**Eugene** referred to original proposal and wishes to see a better motivation for this language feature encouraging use of “plain English” to simplify the use of Scala as practice oriented language. He believes that it could be critical how this SIP can be improved. During the recent conference, organized by Facebook, he spoke with typescript guys that are developing idiomic solutions that would benefit typescript and javascript and allow community users to give their inputs.
+Refers to the paper “Denotation” he linked in a proposal, that is not enough for Scala, but a good start.
+
+**Jorge** is getting back discussion on voting on this proposal and he mentioned that Josh insisted on more examples and suggestions on motivation of this SIP.
+
+**Eugene** wanted to add more syntax (map and flatmap), but Martin opposed to that saying that Scala is quite serious program and needs more reason to add any additional syntax to it. **Martin** would like to see more widespread use of common attic constructs and Libraries, and before doing that, he wouldn’t consider any further change. **Sebastian** agrees with Martin and says that he doesn’t really understand Josh’s and Eugene’s proposal. **Iulian** agrees that the proposal is quite complicated and he wonders how it can be useful. He believes that it is an interesting research direction, but that it needs more users feedbacks in aim to be included in the Scala, therefore questioning if the proposal should be numbered in the current form. Seth and Adriaan agree with Martin and Iulian.
```

Comment:
"common attic constructs" seems incorrect. Must be misheard... (Not sure what it's supposed to be)

Reply:
Can someone go back and listen to the recording to figure out what was said here? This needs to be resolved before this is merged.

Reply:
perhaps "common attic" is "comonadic"

Reply:
Common Attic is frequently used words with roots from ancient Greek. Like monad.